### ISSUE TITLE
changelog display looks broken for "BREAKING CHANGES"

### ISSUE THREAD

**Issue by bugvillage** [ohmyzsh/ohmyzsh#10163]
### Describe the bug

I called recently  `.  tools/changelog.sh`  because I deliberately missed the last update for other reasons - with a quite ugly result (text screenshot in form below)

oh-my-zsh status:

branch: master

last commit: af271c9e

### Steps to reproduce

run  `. tools/changelog.sh` in oh-my-zsh git working directory

### Expected behavior

nicely formatted outout like in other sections (Features, Bug fixes, Other changes)

### Screenshots and recordings

"text screenshot"
```
~/.oh-my-zsh (git)[master] % . tools/changelog.sh                                                                               (master|…4)
master

BREAKING CHANGES:

 - 5a4159c [git]             The
 - been                   Renamed
 - command.                   For
 - conflicts                   With
 - for                   Consistency.
 - `gmt`                   Alias
 - `gmt`                   Alias
 - `gmtl`,                   And
 - `gmtvim`                   Has
 - has                   Been
 - Mapping                   Tools
 - renamed                   To
 - that,                   The
 - the                   Generic
 - to                   `gmtlvim`

Features:

 - 4d5bfe4 [autojump]        Add new Homebrew default path on M1 Macs (#9637)
 - 7336ebc [changelog]       Change style of `BREAKING CHANGES` header
 - a15ac80 [gcloud]          Add Mac M1 Homebrew path (#10143)
 - a5850d4 [git]             Change from commiter to author in `git log` aliases (#9670)
 - 53b54f5 [git]             Guess main branch name also from remotes (#10158)
 - af271c9 [git-auto-fetch]  Add date to git-auto-fetch log file (#10021)
 - c47ac2d [mlh]             Let users configure the official MLH theme (#9997)
 - 1b03896 [pyenv]           Don't load pyenv-virtualenv with `ZSH_PYENV_VIRTUALENV=false`
 - 823be4e [pyenv]           Silence bad config warning with `ZSH_PYENV_QUIET=true`
 - 4a69ee5 [ssh-agent]       Allow specifying absolute path to `identities`
 - 5a9d955 [ssh-agent]       Allow using external helper to ask for passwords (#7631)
 - 7692881 [ssh-agent]       Only start ssh-agent once (#5359)

Bug fixes:

 - 97b4ab7 [chruby]          Fix typo in test for Homebrew path (#9887)
 - 21b385e [frontend-search] Fix codepen.io search (#10157)
 - 0c590ab [git]             Fix `gbda` alias when there are no merged branches (#10005)
 - 5a4159c [git]             Rename `git mergetool` aliases to `gmtl*` (#9154)
 - ab8b991 [git-auto-fetch]  Cancel fetch if we don't have permission over git folder (#10010)
 - 19710a2 [pyenv]           Do not warn if PYENV_ROOT is undefined (#10162)
 - c08fb77 [pyenv]           Properly load pyenv shims and warn of broken configuration
 - 1903250 [sudo]            Allow different $EDITOR settings and fix zsh-syntax-highlighting redraw

Other changes:

 - 3f0672f [ssh-agent]       Refactor: Clean up and reorganize code
 - a1847dc [sudo]            Style: Apply main style guide indentation

```

### OS / Distro

Linux / Debian 10

### Zsh version

5.7.1

### Terminal emulator

Screen 4.06.02 running in  GNOME Terminal 3.36.2 using VTE 0.60.3 +BIDI +GNUTLS +ICU +SYSTEMD

### Additional context

I called the `tools/changelog.sh` manually because for other reasons I purposely missed the update and the changelog.

**Comment by mcornella** [comment]
You're not supposed to run it with `.`, you should run `omz changelog` instead. Does it happen if you do it that way?

**Comment by bugvillage** [comment]
with `omz changelog` I get a better format:

```
BREAKING CHANGES:                                                                                                                                           
                                                                                                                                                            
 - 5a4159c [git]             The `gmt` alias conflicts with the Generic Mapping Tools command. For that, the `gmt` alias has been renamed to `gmtl`, and `gm
tvim` has been renamed to `gmtlvim` for consistency.   
```
(line break caused by terminal width - because it is quite long for a first line in a git commit comment)

Thank's for taking the time to look at this, I will close this issue now.

**Comment by mcornella** [comment]
> (line break caused by terminal width - because it is quite long for a first line in a git commit comment)

This is fixed in 37a60eebc107c08047d4b5716c1c1760104aa25d, you can `omz update` and next time breaking changes are detected you'll get this nicer format:

![Captura de pantalla de 2021-09-07 18-10-43](https://user-images.githubusercontent.com/1441704/132377311-85b8b455-9a92-4a86-b604-785cb2e5d93d.png)
